api/openapi_server/openapi/schemas/_index.yaml

```diff
@@ -27,25 +27,25 @@ ApplicantType:
 HousingProgram:
   type: object
   properties:
-    program_name:           
+    program_name:
       title: program_name
       type: string
     service_provider:
-      $ref: '#/ServiceProvider'
+      $ref: "#/ServiceProvider"
```

Comment:
can you undo these changes? can move the linting/style check stuff into a separate PR.